webrtc-stats.html

```diff
@@ -1859,23 +1841,56 @@ <h2>
               </dt>
               <dd>
                 <p>
-                  Only valid for video. It represents the nominal FPS value before the degradation
+                  Represents the nominal FPS value before the degradation
                   preference is applied. It is the number of complete frames in the last second.
                   For sending tracks it is the current captured FPS and for the receiving tracks it
                   is the current decoding framerate.
                 </p>
               </dd>
+            </dl>
+          </section>
+        </div>
+      </section>
+      <section id="vsstats-dict*">
+        <h3>
+          <dfn>RTCVideoSenderStats</dfn> dictionary
```

Comment:
We have these hierarchies (and similar for audio and receive):

`RTCMediaStreamTrackStats <- RTCVideoStreamTrackStats <- RTCVideoSenderStats <- RTCLocalVideoStreamTrackStats`

In other words

`Track <- [Audio/Video]Track <- [Audio/Video][Sender/Receiver] <- Local[Audio/Video]Track`

It gets a bit confusing about going back and forth between track language and sender/receiver language.

The way I see it, it's sender/receiver all the way down. The `RTCLocal[Audio/Video]StreamTrackStats` are the only special case where now we're not talking about a _sender_, now it changes to an _attachment_ of a track (it's not even a track-stat, since attaching the same track multiple times yields different attachment stats). Which really is a subview of the sender stats for the duration a track is attached.

Lacking a better word that can be used to refer to both a sender and a receiver than "SenderReceiver" (and avoiding confusion with `RTCRtpTransceiver`), I would name them:
```
RTCSenderReceiverStats (RTCMediaStreamTrackStats)
RTC[Audio/Video]SenderReceiverStats (RTC[Audio/Video]StreamTrackStats)
RTC[Audio/Video][Sender/Receiver]Stats (same name as before)
RTCSender[Audio/Video]TrackAttachmentStats (RTCLocal[Audio/Video]StreamTrackStats)
```

In other words

`SenderReceiver <- [Audio/Video]SenderReceiver <- [Audio/Video][Sender/Receiver] <- [Audio/Video]TrackAttachment`

But renames can be done in separate PRs.